### PR TITLE
docs: add instructions for debugging tests via terminal

### PR DESCRIPTION
**Pull request by joshnuss**
Adds example of running `vitest` with debugger in the shell (without an IDE)

### CODE REVIEW

File: docs/guide/debugging.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ title: Debugging | Guide
 
 # Debugging
 
+## Terminal
+
+To debug a test file in your shell, add a `debugger` statement anywhere in your code, and then run `ndb`:
+
+```sh
+# install ndb globally
+npm install -g ndb
```
Comment (dammy001) [inline docs/guide/debugging.md:13]:
```suggestion
npm install -g ndb

# alternatively, with yarn
yarn global add ndb
```

Comment (dammy001) [inline docs/guide/debugging.md:13]: Can we add a reference to ndb docs?

Comment (joshnuss) [reply docs/guide/debugging.md:13]: Maybe we should mentioned `pnpm` too? I'm never sure if it's better to include all three, or do folks know?

Comment (joshnuss) [reply docs/guide/debugging.md:13]: ndb doesn't have any docs, but we can link to the github repo. It has a helpful readme.

**Comment by dammy001** [comment]
@sheremet-va check it

**Review by sheremet-va — APPROVED** [review]

**Comment by 1EDExg0ffyXfTEqdIUAYNZGnCeajIxMWd2vaQeP** [comment]
Sadly, ndb is unmaintaned and doesn't work with node 19. It shouldn't be recommended.

**Comment by joshnuss** [comment]
Thanks @1EDExg0ffyXfTEqdIUAYNZGnCeajIxMWd2vaQeP,

I've opened a PR #2869, to revert this change.

**Pull request merged**